backend/controllers/auth.js

```diff
@@ -1,20 +1,32 @@
 import { HttpError } from "../helpers/HttpError.js";
 import { createApiResponse } from "../helpers/createApiResponse.js";
 import { prisma } from "../lib/db.js";
+import jwt from "jsonwebtoken";
+import bcrypt from "bcrypt";
 
 const COOKIE_MAX_AGE_MILISECONDS = 60 * 60 * 1000;
+const { JWT_SECRET } = process.env;
 
 const register = async (req, res, next) => {
   try {
     const { username, password } = req.body;
+
+    const hashPassword = await bcrypt.hash(password, 10);
+
     const user = await prisma.user.create({
       data: {
-        password,
+        password: hashPassword,
```

Comment:
Please migrate the user model to use a more fitting name for the hash field. something like `passwordHash`  will work just fine.

Reply:
I have doubts about this, whether it is really necessary